cmd/boots/main.go

```diff
@@ -97,13 +140,162 @@ func main() {
 		}
 	}()
 
-	mainlog.Info("serving tftp")
-	go ServeTFTP()
-	mainlog.Info("serving dhcp")
-	go ServeDHCP()
+	var nextServer net.IP
+	var httpServerFQDN string
```

Comment:
small nit: it would be nice if these were define close to when they were actually used. So nextServer right before the `if cfg.RemoteTFTPAddr` check and httpServerFQDN right before the `if cfg.remoteiHTTPAddr` with the other related vars.

Reply:
update